debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlHistoryRecordComparator.java

```diff
@@ -145,4 +143,79 @@ else if (recordedGtidSetStr != null) {
         // The binlog coordinates are the same ...
         return true;
     }
+
+    public static class BinlogFilename implements Comparable<BinlogFilename> {
+        final private String baseFilename;
```

Comment:
```suggestion
        final private String baseName;
```
I'd use the "base name" term for consistency with the [MySQL documentation](https://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#option_mysqld_log-bin) and the Unix terminology in general.